.github/workflows/style.yaml

```diff
@@ -0,0 +1,74 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  pull_request:
+    branches: [main]
+    paths: ["**.[rR]", "**.[qrR]md", "**.[rR]markdown", "**.[rR]nw", "**.[rR]profile"]
+
+name: Sjekk kodestil
+
+jobs:
+  style:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Setup R
+        uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          extra-packages: any::styler
+          needs: styler
+
+      - name: Document
+        run: roxygen2::roxygenise()
+        shell: Rscript {0}
+
+      - name: Install package
+        run: R CMD INSTALL .
+
+      - name: Enable styler cache
+        run: styler::cache_activate()
+        shell: Rscript {0}
+
+      - name: Determine cache location
+        id: styler-location
+        run: |
+          cat(
+            "location=",
+            styler::cache_info(format = "tabular")$location,
+            "\n",
+            file = Sys.getenv("GITHUB_OUTPUT"),
+            append = TRUE,
+            sep = ""
+          )
+        shell: Rscript {0}
+
+      - name: Cache styler
+        uses: actions/cache@v3
+        with:
+          path: ${{ steps.styler-location.outputs.location }}
+          key: ${{ runner.os }}-styler-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-styler-
+            ${{ runner.os }}-
+
+      - name: Style
+        run: styler::style_pkg(style = rapwhale::rapwhale_style)
+        shell: Rscript {0}
+
+      - name: Verify Changed files
+        uses: tj-actions/verify-changed-files@v16
+        id: verify-changed-files
+
+      - name: Run step only when any of the above files change.
+        if: steps.verify-changed-files.outputs.files_changed == 'true'
+        run: |
+          echo "::error::Filer med feil formatering: ${{ steps.verify-changed-files.outputs.changed_files }}"
+          exit 1
```

Comment:
Kan kanskje alt erstattast med følgjande `run`-steg:

    git diff --exit-code

Kommandoen vil gje exit-verdi lik 1 viss nokon filer er endra. Og utdata vil jo vera resultatet av diff-kommandoen, som vil visa både kva filer er feilformaterte og korleis dei er feilformaterte. Og det er jo nettopp den informasjon me treng. Verdt eit forsøk? Og så slepp me å vera avhengig av kode i eit depot me ikkje har kontroll over.

Reply:
Kanskje òg med `--ws-error-highlight=all`, for å visa kor det er endringar i blankteikn.

Reply:
Skal testa. 👍

Reply:
Ser ut til å funka. Prøv no med `--patch-with-stat --color=always` i tillegg.

Reply:
Lekkert. Effekten av `--ws-error-highlight=all` vert ikkje synleg, men det er vel GitHub sin feil. Bør bli ståande, så kanskje vert det fiksa ein gong i framtida.